semeio/jobs/correlated_observations_scaling/job_config.py

```diff
@@ -154,6 +160,11 @@ def build_schema():
                     }
                 },
             },
+            "EXPORT_DIR": {
+                MK.Type: types.String,
+                MK.Required: False,
+                MK.ElementValidators: (_path_exists,),
```

Comment:
Should it even be a folder perhaps?

Reply:
What do you mean?

Reply:
It is only validated that it exists... I guess we want it to be a folder?

Reply:
Right. Good point.